x-pack/plugins/reporting/server/routes/common/generate/request_handler.ts

```diff
@@ -106,11 +114,64 @@ export class RequestHandler {
     return report;
   }
 
-  public async handleGenerateRequest(
-    exportTypeId: string,
-    jobParams: BaseParams,
-    counters: Counters
-  ) {
+  public getJobParams(): BaseParams {
+    let jobParamsRison: null | string = null;
```

Comment:
Without this helper, this code would have had to be duplicated in `x-pack/plugins/reporting/server/routes/internal/management/jobs.ts` and `x-pack/plugins/reporting/server/routes/public/jobs.ts`